docs/framework/unmanaged-api/debugging/icordebugprocess4-interface.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "ICorDebugProcess4 Interface"
+ms.date: "02/07/2019"
+api_name:
+  - "ICorDebugProcess4"
+api_location:
+  - "mscordbi.dll"
+api_type:
+  - "COM"
+f1_keywords:
+  - "ICorDebugProcess4"
+helpviewer_keywords:
+  - "ICorDebugProcess4 interface [.NET Framework debugging]"
+topic_type:
+  - "apiref"
+author: "hoyosjs"
+ms.author: "juhoyosa"
+---
+# ICorDebugProcess4 Interface
+Provides support for out of process execution control.
+
+## Methods
+
+| Method                                                                                                                    | Description                                                                                             |
+| ------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------- |
+| [ProcessStateChanged](../../../../docs/framework/unmanaged-api/debugging/icordebugprocess4-processstatechanged-method.md) | Notifies the ICorDebug pipeline that the out of process debugger is continuing the debugee's execution. |
```

Comment:
Easier to use the shorter path. The long paths are due to the script that migrated the content from MSDN

```suggestion
| [ProcessStateChanged](icordebugprocess4-processstatechanged-method.md) | Notifies the ICorDebug pipeline that the out of process debugger is continuing the debugee's execution. |
```